docs/docs/usage/advanced.md

```diff
@@ -240,3 +240,15 @@ This hook wraps the command `pdm export` along with any valid argument. It can b
       args: ['-o', 'requirements.txt', '--without-hashes']
       files: ^pdm.lock$
 ```
+
+### Check `.pdm.lock` is up to date
+
+This hook wraps the command `pdm lock --check` along with any valid argument. It can be used as a hook (e.g., for CI) to ensure that whenever `pyproject.toml` has a dependency added/changed/removed, that `.pdm.lock` is also up to date.
+
+```yaml
+# export python requirements
```

Comment:
This line seems copied from other section, need change

Reply:
https://github.com/pdm-project/pdm/pull/1475/commits/70809b43c16dae58f970a5af6057044a01e19c7a